prisma/schema.prisma

```diff
@@ -0,0 +1,26 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider  = "postgresql"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+}
+
+model User {
+  address        String        @id @unique
+  last_connected DateTime?     @db.Timestamptz(6)
+  Transaction    Transaction[]
+}
+
+model Transaction {
+  transaction_hash String    @id
+  user_address     String
+  chain_id         Decimal?  @db.Decimal
+  from             String?
+  to               String?
+  amount           String?
```

Comment:
`amount` is going to be the USD amount, or something else? (thinking about the data type)

Reply:
we could possibly store the token amount, and the usd price of the given amount as well.